server/prpc/server.go

```diff
@@ -37,6 +37,10 @@ var (
 	// exposeHeaders lists the whitelisted non-standard response headers that the
 	// client may accept.
 	exposeHeaders = strings.Join([]string{prpccommon.HeaderGRPCCode}, ", ")
+
+	//NoAuthenticator should be used with a Server to explicitly specify it wont use any Auth
```

Comment:
nits: spaces after "//", period after "panic", missing apostrophe in "won't", "Auth" isn't a keyword (did you mean, "Authenticator"), and please reword to mention the `RegisterDefaultAuth` function name, something like:
NoAuthenticator can be used with RegisterDefaultAuth or Server.Authenticator to explicitly specify that your Server won't apply any authentication to incoming connections.